model/atmosphere/diffusion/tests/diffusion_stencil_tests/test_apply_diffusion_to_theta_and_exner.py

```diff
@@ -82,10 +82,8 @@ def reference(
     def input_data(self, grid):
         # TODO [halungge]: understand why values do not verify intermittently
         # error message contained in truly_horizontal_diffusion_nabla_of_theta_over_steep_points_numpy
+        # TODO [nfarabullini]: set this as another marker
         pytest.xfail("fix start index issue")
```

Comment:
Isn't that the `skip_value_error`? Or is the error even still there, because I can't see any error message in `truly_horizontal_diffusion_nabla_of_theta_over_steep_points_numpy` or not sure how to read @halungge 's TODO.